vips/vip-169/vip-169.ts

```diff
@@ -0,0 +1,35 @@
+import { ProposalType } from "../../src/types";
+import { makeProposal } from "../../src/utils";
+
+const HAY_REWARDS_DISTRIBUTOR = "0xA31185D804BF9209347698128984a43A67Ce6d11";
+const SD_REWARDS_DISTRIBUTOR = "0xBE607b239a8776B47159e2b0E9E65a7F1DAA6478";
+const MARKET_BNBx = "0x5E21bF67a6af41c74C1773E4b473ca5ce8fd3791";
+const MARKET_HAY = "0xCa2D81AA7C09A1a025De797600A7081146dceEd9";
+
+export const vip169 = () => {
+  const meta = {
+    version: "v2",
+    title: "VIP-163 Stop HAY and SD Rewards",
+    description: `Stop HAY and SD rewards`,
+    forDescription: "I agree that Venus Protocol should proceed with this proposal",
+    againstDescription: "I do not think that Venus Protocol should proceed with this proposal",
+    abstainDescription: "I am indifferent to whether Venus Protocol proceeds or not",
+  };
+
+  return makeProposal(
+    [
+      {
+        target: HAY_REWARDS_DISTRIBUTOR,
+        signature: "setLastRewardingBlocks(address[],uint32[],uint32[])",
+        params: [[MARKET_HAY], ["32713205"], ["32713205"]],
+      },
+      {
+        target: SD_REWARDS_DISTRIBUTOR,
+        signature: "setLastRewardingBlocks(address[],uint32[],uint32[])",
+        params: [[MARKET_BNBx], ["32713205"], ["32713205"]],
+      },
```

Comment:
Rewards for HAY and BNBx were enabled on the [VIP-163](https://app.venus.io/#/governance/proposal/163), which was executed on the block [31394389](https://bscscan.com/tx/0xa11e772311fa4b2efe76445b64fc0a642c9fd469b3df6bd4464738927beadcda).

HAY rewards must be enabled for 28 days. That means for 806400. So, the last block with rewards in the HAY market should be `31394389 + 806400 = 32200789`

BNBx rewards must be enabled for 30 days. That means for 864000. So, the last block with rewards in the BNBx market should be `31394389 + 864000 = 32258389`

The same calculation should be done for the testnet environment, taking into account that the VIP where the rewards were enabled was [this one](https://testnet-cf.venus.io/#/governance/proposal/280), expected on the block [32871078](https://testnet.bscscan.com/tx/0xb06be6ed9813e4e4c675702bd74bb0f75c897c25365d9eea3387520c9ff39257)